src/bundle.c

```diff
@@ -723,6 +728,35 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	to_install_files = filter_out_existing_files(to_install_files);
 	grabtime_stop(&times);
 
+	/* Step 1.5: Now that we know the bundle is valid, check if we have enough space */
+	ptr = list_head(to_install_bundles);
+	while (ptr) {
+		subman = ptr->data;
+		ptr = ptr->next;
+
+		struct bundle_result *bundle = NULL;
+		bundle = calloc(sizeof(struct bundle_result), 1);
+		/* copy relevant information over for future use */
+		strncpy(bundle->bundle_name, subman->component, BUNDLE_NAME_MAXLEN - 1);
+		bundle->topsize = subman->contentsize;
+		/* do a deep copy of the includes list */
+		bundle->includes = list_deep_clone_strs(subman->includes);
+		bundle_info = list_prepend_data(bundle_info, bundle);
+		bundle_size += calculate_size(subman->component, bundle_info, 0);
+	}
+
+	fs_free = get_available_space("/usr/");
+
+	/* Add 10% to bundle_size as a 'fudge factor' */
+	if (bundle_size + (bundle_size * .10) > fs_free && !skip_diskspace_check) {
+		ret = EBUNDLE_INSTALL;
+		fprintf(stderr, "Error: Bundle too large by %ldM.\n", (bundle_size - fs_free) / 1000 / 1000);
```

Comment:
You should divide by 1024 instead of 1000 if you want the size in M

Reply:
Agreed, but 1000 was in use before I copy/pasted this function. Search.c uses 1000 for example.

Maybe this should be MiB instead of MB? Should we report in MiB or MB?

Reply:
The intention for search was to report in MB, not MiB, this may be a different use-case though.